source/code/go/src/plugins/oms.go

```diff
@@ -319,12 +342,14 @@ func InitializePlugin(pluginConfPath string) {
 	pluginConfig, err := ReadConfiguration(pluginConfPath)
 	if err != nil {
 		Log("Error Reading plugin config path : %s \n", err.Error())
+		SendException(err)
```

Comment:
You might want to send the string that you are logging below this line as well. Else you dont know from where all these errors are coming from and hard to match to code.

Reply:
Sounds good